### PR TITLE
Added alias for setting CommandTimeout

### LINKED ISSUE #12: Need ability to specify command timeout
I am having issues with timeouts because the default value for SqlCommand.CommandTimeout is 30 seconds. There is currently no way to set it through the Cake.SqlServer API.

### PR DESCRIPTION
**Pull request by kmc059000**
Closes #12

### CODE REVIEW

**Comment by kmc059000** [comment]
I used a new alias in order to avoid having to change any existing aliases and keep backwards compatibility. 

If users do not call `SetCommandTimeout()` then the default will continue to be used.

File: src/Cake.SqlServer/SqlServerAliasesImpl.cs [hunk 1]
```diff
@@ -167,5 +167,22 @@ internal static SqlConnection OpenSqlConnection(ICakeContext context, String con
                 throw;
             }
         }
+
+        internal static void SetCommandTimeout(ICakeContext context, int commandTimeout)
+        {
+            CommandTimeout = commandTimeout;
```
Comment (trailmax) [inline src/Cake.SqlServer/SqlServerAliasesImpl.cs]: Please add a log message to `context.Log.Debug($"Default SQL timeout have been changed to {commandTimeout}")`

**Comment by trailmax** [comment]
By the way, don't worry about failed build - gitversion for some reason does not like pull requests. I need to fix the build script for that.

File: src/Cake.SqlServer/SqlServerAliasesImpl.cs (hunk 1 above)
Comment (kmc059000) [inline src/Cake.SqlServer/SqlServerAliasesImpl.cs]: This is resolved now.

**Comment by trailmax** [comment]
PR Accepted. Many thanks for your contribution! :+1: :cake: :champagne:

**Comment by trailmax** [comment]
After some considerations I've renamed `SetCommandTimeout` to `SetSqlCommandTimeout` to avoid possible namespace clashes with other Cake plugins. 

And NuGet v1.4.0 is published now, about to be indexed by NuGet.org: https://www.nuget.org/packages/Cake.SqlServer/1.4.0

**Comment by kmc059000** [comment]
That makes sense. Thank's for publishing to nuget, I'll keep a lookout for it.

**Pull request merged**